modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -89,6 +99,34 @@ public void createDeployableProjectModelWithAdditionalDependenciesInAPlugin() th
     assertThat(descriptor.getArtifactId(), is("derby"));
   }
 
+
+  @Test
+  @Issue("W-14998254")
+  @Description("When pom has invalid GAV and no parent pom, then MuleRuntimeException will be the thrown.")
+  public void createDeployableProjectModelWithInvalidGAVAndMissingParentPom() throws Exception {
+    expectedException.expect(MuleRuntimeException.class);
+    getDeployableProjectModel("apps/lightweight/test-app-missing-gav", null);
+  }
+
+
+  @Test
+  @Issue("W-14998254")
+  public void createDeployableProjectModelWithGAVPresentInParentPom() throws Exception {
+
+    File artifact =
+        new File(getMavenConfig().getLocalMavenRepositoryLocation(),
+                 "org/mule/test/api-app/1.0.0/");
+
+    PluginFileMavenReactor pluginFileMavenReactor =
+        new PluginFileMavenReactor(artifact, "org.mule.test", "api-app", "1.0.0");
+
+    DeployableProjectModel deployableProjectModel =
+        getDeployableProjectModel("apps/lightweight/test-app-missing-gav-valid-parent", pluginFileMavenReactor);
+    MatcherAssert.assertThat(deployableProjectModel.getDescriptor().getVersion(), Matchers.is("1.2.0-POM"));
+    MatcherAssert.assertThat(deployableProjectModel.getDescriptor().getGroupId(), Matchers.is("test"));
+    MatcherAssert.assertThat(deployableProjectModel.getDescriptor().getArtifactId(), Matchers.is("api-app"));
```

Comment:
Use static imports as per the [coding conventions](https://docs.google.com/document/d/1N2Axq3ABJjYmj9BjfYQkvJfF0odHQnlb8MAdVQydOWI).